spaceship/lib/spaceship/connect_api/models/app_store_version_localization.rb

```diff
@@ -15,20 +15,14 @@ class AppStoreVersionLocalization
       attr_accessor :support_url
       attr_accessor :whats_new
 
-      attr_accessor :app_screenshot_sets
-      attr_accessor :app_preview_sets
```

Comment:
Paranoia: Have you checked that no one is calling these? I guess `rubocop` would complain but I don't know if it can check that too.

Reply:
I've checked ✅ No direct calls to these accessors.